lib/matplotlib/_mathtext.py

```diff
@@ -2841,6 +2859,40 @@ def binom(self, s, loc, toks):
         return self._genfrac('(', ')', 0.0, self._MathStyle.TEXTSTYLE,
                              num, den)
 
+    def _genset(self, state, value, symbol, shift):
+        thickness = state.font_output.get_underline_thickness(
+            state.font, state.fontsize, state.dpi)
+
+        value.shrink()
+
+        cvalue = HCentered([value])
+        csymbol = HCentered([symbol])
+        width = max(cvalue.width, csymbol.width)
+        cvalue.hpack(width, 'exactly')
+        csymbol.hpack(width, 'exactly')
+
+        metrics = state.font_output.get_metrics(
+            state.font, mpl.rcParams['mathtext.default'],
+            r'\leftrightarrow', state.fontsize, state.dpi)
+
+        # Shift so the symbol sits in the same vertical position
+        shift_amount = csymbol.height/2 - (metrics.ymax + metrics.ymin)/2
+        if shift:
```

Comment:
given that this is tightly coupled to over/under (as the vlist order is also set here), I'd just rename `shift` to e.g. `overunder` which is either "over" or "under" for readability.